WDscholia/scholia
# Add formats, fix bugs in work export

Fixes #1971  
Fixes #2083  
See #1594  

### Description

  - Two additional formats are listed among the export options: EndNote .enw files, and Wikipedia templates.

![Screenshot_20221026_172908](https://user-images.githubusercontent.com/14018963/198069181-a1600067-6ba9-4f26-bf67-1927c5890e59.png)
![Screenshot_20221026_172923](https://user-images.githubusercontent.com/14018963/198069223-a964f9d4-7d15-408f-bfdb-829ae45af6cd.png)

  - Empty strings are no longer treated as valid values in QuickStatements output.

### Caveats

* [x]  This change requires new dependencies (please list): JS-only, two new Citation.js plugins (enw and wikipedia-templates)

[Bundle configuration](https://juniper-coat.glitch.me/):  

![Screenshot_20221026_171512](https://user-images.githubusercontent.com/14018963/198069086-91151901-ca1b-43d6-881a-0d4047ae0e3b.png)

*If you make changes to the Python code*
  

### Testing

* Go to http://127.0.0.1:8100/work/Q112075162/export
* Select ENW, check output
* Select Wikipedia Templates, check output

### Checklist
